eng/Packages.Data.props

```diff
@@ -391,6 +391,7 @@
     <PackageReference Update="WindowsAzure.ServiceBus" Version="5.1.0" />
     <PackageReference Update="xunit.runner.visualstudio" Version="2.4.2" />
     <PackageReference Update="xunit" Version="2.4.2" />
+    <PackageReference Include="xunit.extensibility.core" Version="2.9.3" />
```

Comment:
Current generation Azure libraries are required to use NUnit for tests.   XUnit is available only to legacy packages and those with approved special needs.

This dependency should be removed.

Reply:
ok will do.